content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,658 @@
+---
+title: "Build Edge Artifacts"
+metaTitle: "Build Edge Artifacts"
+metaDescription: "Learn how to build installer ISO and provider OS images using Spectro Cloud's CanvOS utility."
+icon: ""
+hideToC: false
+fullWidth: false
+hideToCSidebar: false
+hiddenFromNav: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from "shared/components/common/PointOfInterest";
+
+# Build Edge Artifacts
+Palette's Edge solution requires Edge hosts to be ready with the required dependencies and [user data](/clusters/edge/edge-configuration/installer-reference) configurations before deploying a Kubernetes cluster. An Edge host requires the following artifacts for a successful deployment:
+
+<br />
+
+* **Edge installer ISO image** - This bootable ISO image installs the necessary dependencies and configurations on a bare host machine. During installation, the host machine will boot from the Edge installer ISO, partition the disk, copy the image content to the disk, install the Palette Edge host agent and metadata, and perform several configuration steps. These configuration steps include registering the host with Palette, setting user privileges, and configuring network or security settings. 
+<br />
+
+* **Provider OS image** - This is a [Kairos](https://kairos.io/)-based image. This image installs an immutable Operating System (OS) and software dependencies compatible with a specific Kubernetes version at runtime, i.e., during the cluster deployment.  
+
+
+In this guide, you will use Spectro Cloud's utility, [CanvOS](https://github.com/spectrocloud/CanvOS), to create an Edge installer ISO image and two provider OS images - one compatible with K3s v1.24.6 and another compatible with K3s v1.25.2. It builds two provider OS images, so you can use either of them that matches the Kubernetes version in your cluster profile.
+
+
+The diagram below shows the high-level steps to build the artifacts and push to an image registry.
```

Comment:
The diagram is great but step 6 is what I worry could cause confusion. That's not done in this how-to, so I think it's best to remove it. Also, a USB is not the only way to load an Edge host, you've seen that with VMware.